### DISCUSSION TITLE
I expect the editor to display the results of a replace command in real-time, similar to the behavior in VSCode.

### DISCUSSION THREAD

**Discussion by lianghengyuan** [JetBrains/ideavim#756] [Q&A]
In VSCode, when I execute a replace command, the editor shows the results of the replacement in real-time, making it easy to see the changes immediately. However, in IdeaVim, I've observed that the results of a replace command are not displayed in real-time, and I need to manually navigate through the document to visualize the changes.

**Comment by citizenmatt** [comment]
IdeaVim follows Vim's behaviour, rather than VSCode's. Do you see similar behaviour in Vim, or know a Vim plugin that gives the same?

**Reply by UyCode** [reply]
yes, it is not the behavior of vim, but NeoVim, VScode vim plugin are implemented the feature, we expecting Ideavim support this feature

**Reply by Mati20041** [reply]
Hi, I stepped into this thread because I'm looking for a way to achieve something similar. I've found out that this vim plugin does exactly that what I'd really like to have in ideavim https://github.com/markonm/traces.vim
